csvkit/utilities/csvjson.py

```diff
@@ -110,39 +144,48 @@ def dump_json(data, newline=False):
                 feature['type'] = 'Feature'
                 properties = OrderedDict()
                 geoid = None
+                geo_type = 'Point'
                 lat = None
                 lon = None
+                coords = None
 
                 for i, c in enumerate(row):
                     if i == lat_column:
+                        if not c:
```

Comment:
I'm wondering if a '0' is read as an integer `0` instead of the string `'0'`, then we should check `if c is None` instead.